src/SFA.DAS.Commitments.Application/Queries/GetEmployerAccountSummary/GetEmployerAccountSummaryValidator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using FluentValidation;
+
+namespace SFA.DAS.Commitments.Application.Queries.GetEmployerAccountSummary
+{
+    public sealed class GetEmployerAccountSummaryValidator : AbstractValidator<GetEmployerAccountSummaryRequest>
```

Comment:
Should this be using an Api Type as it's return value? We seem to be doing that on other Handler responses